### PR TITLE
Pasted palindrome code and tests

### PR DESCRIPTION
**Pull request by lyudarkim**
Please review!! Thank you!!

### CODE REVIEW

File: functions/palindrome_number.py [hunk 1]
```diff
@@ -1,2 +1,12 @@
 def palindrome_number(x):
-    pass
+    temp = x
+    reverse = 0
+    while(x>0):
```
Comment (suyon02kim) [inline functions/palindrome_number.py]: Please change the spacing so that there are spaces in between!

File: functions/palindrome_number.py [hunk 2]
```diff
@@ -1,2 +1,12 @@
...
+        dig = x%10
```
Comment (suyon02kim) [inline functions/palindrome_number.py]: Same over here!

File: functions/palindrome_number.py [hunk 3]
```diff
@@ -1,2 +1,12 @@
...
+        reverse = reverse*10 + dig
+        x = x//10
+
+    if temp == reverse:
```
Comment (suyon02kim) [inline functions/palindrome_number.py:10]: Hi Lyuda :) We are almost done with class!!!

File: tests/palindrome_number_test.py [hunk 4]
```diff
@@ -1,2 +1,35 @@
...
+    n = 99
+
+    # Act
+    answer = palindrome_number(99)
+
+    # Assert
+    assert answer == True
+
+
+def test_returns_true_for_151():
```
Comment (suyon02kim) [inline tests/palindrome_number_test.py:16]: I think adding an edge case will be good here! (But don't really do it)

**Review by suyon02kim — COMMENTED** [review]
Please thank you

File: tests/palindrome_number_test.py (hunk 4 above)
Comment (lyudarkim) [reply tests/palindrome_number_test.py:16]: Ok, I won't do it! :)
